<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DactQueryWindow</class>
 <widget class="QWidget" name="DactQueryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="filterLabel">
       <property name="text">
        <string>Filter:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="filterLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="detailLabel">
       <property name="text">
        <string>Count:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="attributeComboBox">
       <item>
        <property name="text">
         <string>cat</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>pos</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>rel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>root</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="resultsTableWidget">
     <column>
      <property name="text">
       <string>result</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>count</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>percentage</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="percentageCheckBox">
     <property name="text">
      <string>Show Percentages</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
